.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -36,7 +36,7 @@ or the Traffic Control Users mailing list (send an email to users-subscribe@traf
 <!-- Do not submit security vulnerabilities or support requests here - see above -->
 ## This Bug Report affects these Traffic Control components:
 <!-- delete all those that don't apply -->
-- Traffic Control Cache Config (T3C, formerly ORT)
+- Traffic Control Cache Config (t3c, formerly ORT)
```

Comment:
> The application is `t3c`, not "T3C,"...

It seems like this should be mono-spaced?

Reply:
Sure. Done.